engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubConnection.java

```diff
@@ -136,4 +143,11 @@ private URIBuilder appendToPath(final URIBuilder uriBuilder, final String pathSe
 
         return uriBuilder.setPath(pathSegment);
     }
+
+    private class DataHubRequestRetryHandler extends DefaultHttpRequestRetryHandler {
+        DataHubRequestRetryHandler() {
+            super(3, false,
+                    Arrays.asList(InterruptedIOException.class, UnknownHostException.class, SSLException.class));
```

Comment:
Ahhh, get it. It's used multiple times, so it was this or a createRetryHandler method (though I think you might be allowed to reuse a retry handler).